src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,52 +1,147 @@
 package core.basesyntax;
 
-/**
- * <p>Реалізувати свій ArrayList який імплементує інтерфейс List</p>
- */
 public class ArrayList<T> implements List<T> {
+    private int size;
+    private Object[] array = new Object[10];
```

Comment:
10 is the magic number?

Reply:
10 is default capacity of array in ArrayList...